README.md

```diff
@@ -14,6 +14,12 @@ The [contributing documentation](CONTRIBUTING.md) covers licencing and the usual
 
 **Do Not Ever Update Quay Labels Based On Your Local Development Branch. Only from master!**
```

Comment:
Not a review comment but as a note, this will not be an issue when Cincinnati will only look in to git for graph data. It will also reduce the moving